packages/towns-bot-cli/src/index.ts

```diff
@@ -49,29 +61,52 @@ ${yellow('Usage:')}
   towns-bot <command> [options]
 
 ${yellow('Commands:')}
-  ${green('init')} [project-name]    Create a new bot project
-  ${green('update')}                 Update @towns-protocol dependencies to latest versions
+  ${green('init')} [project-name]     Create a new bot project
+  ${green('update')}                  Update @towns-protocol dependencies to latest versions
+  ${green('list-commands')}           List slash commands from a file
+  ${green('update-commands')}         Update slash commands for a bot
 
-${yellow('Options:')}
-  -t, --template <name>   Template to use:
+${yellow('Init Options:')}
+  -t, --template <name>    Template to use:
 ${Object.entries(TEMPLATES)
     .map(
         ([key, template]: [string, Template]) =>
-            `                            ${key} - ${template.description}`,
+            `                             ${key} - ${template.description}`,
     )
     .join('\n')}
-                          Default: quickstart
-  -h, --help              Show this help message
+                           Default: quickstart
+
+${yellow('List Commands Options:')}
+  -f, --file <path>        Path to commands file
+
+${yellow('Update Commands Options:')}
+  -f, --file <path>         Path to commands file
+  -t, --bearerToken <token>  Bearer token for authentication
+  -e, --envFile <path>       Path to .env file (default: .env)
+
+${yellow('Global Options:')}
+  -h, --help               Show this help message
 
 ${yellow('Examples:')}
-  ${cyan('# Create a new bot project with the default template')}
+  ${cyan('# Create a new bot project')}
   towns-bot init my-bot
-
-  ${cyan('# Create an AI bot project')}
   towns-bot init my-ai-bot --template thread-ai
 
-  ${cyan('# Update dependencies in current project')}
+  ${cyan('# Update dependencies')}
   towns-bot update
+
+  ${cyan('# List slash commands')}
+  towns-bot list-commands src/commands.ts
+  towns-bot list-commands --file src/commands.ts
+
+  ${cyan('# Update slash commands (positional arguments)')}
+  towns-bot update-commands commands.ts token123
```

Comment:
Later, we could be fancy and have a `towns-bot auth` that stores the bearer token in keychain, and then we can remove the bearer token argument here